src/main/java/uk/gov/hmcts/juror/api/moj/service/JurorRecordServiceImpl.java

```diff
@@ -698,8 +698,10 @@ public void createJurorContactLog(BureauJwtPayload payload, ContactLogRequestDto
             contactLogRequestDto.getJurorNumber(), payload.getOwner());
         // check whether the current user has permissions to create new contact logs against the currently active
         // juror record
-        JurorPoolUtils.checkOwnershipForCurrentUser(jurorPool, payload.getOwner());
-
+        if (!("400".equals(payload.getOwner()) || jurorPool.getOwner().equals(payload.getOwner()))) {
```

Comment:
I think this if statement might only work for the bureau only. I think for court users this will throw an exception as it'll always be true.

Reply:
There's unit tests for court users working, so I think this works fine? It should be `!(bureauCheck || courtUser == poolOwner)`, which I think is what we're expecting?